src/nupic/bindings/sparse_matrix.i

```diff
@@ -177,7 +178,7 @@ def __init__(self, *args):
 # def getSliceWrap(self, key):
 #   bounds = [ self.nRows(), self.nCols() ]                                                    d = self._getDomain(key, bounds)
```

Comment:
I know this had been commented out, but still the statement on far right looks like it belongs on the next line